awx/main/tasks/jobs.py

```diff
@@ -800,7 +640,9 @@ class RunJob(BaseTask):
 
     model = Job
     event_model = JobEvent
-    event_data_key = 'job_id'
+
+    def __init__(self):
+        super(RunJob, self).__init__()
```

Comment:
shouldn't need this `__init__`, there's also another init method in the callback module that you can lose